README.md

```diff
@@ -314,19 +313,32 @@ const Loader = styled('div')`
 `;
 ```
 
-### With props
-
-It can also be useful to create components with props already applied, like the example below. We recommend using recompose's `withProps` higher-order component to do this.
+### Helpers
 
-**[`withProps` documentation](https://github.com/acdlite/recompose/blob/master/docs/API.md#withprops)**
+A common task with styled components is to configure or map their props. We include a few helpers you can
+optionally include to do this if you want, they are extra and if you don't use them they won't be included in your bundle. There are a few advantages to using the included helpers over a more general solution
+like `recompose`. They automatically forward `refs`, and don't muck with the `as` prop passthrough.
 
 ```jsx
 import styled from 'astroturf';
-import withProps from 'recompose/withProps';
+import { withProps, defaultProps, mapProps } from 'astroturf/helpers';
```

Comment:
so, do `defaultProps` and `withProps` actually make sense? especially in the non-functional form, it seems like you could almost just take `defaultProps` as a 2nd arg to `styled`

Reply:
That's a good point for defaultProps. I do think there is utility in having all three behaviors...judging from our usage withProps being the most used. Idk. Is it worth having defaultProps be different?

Reply:
it looks like our usage of `withProps` mostly could just be `defaultProps`